CHANGELOG.md

```diff
@@ -7,15 +7,18 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ## [Unreleased]
 
 **Please Note**
+- In order for Cumulus to produce reports of files accessed via your distribution API, you must set up S3 access logging on all of your protected buckets. See [documentation on EMS reporting](https://nasa.github.io/cumulus/docs/ems_reporting) for how to correctly set up S3 access logging for your deployment.
 - If you are using the `@cumulus/post-to-cmr` task in your workflow and specifying the metadata format to post using the `cmrFileType` config property, you must update that workflow step to use the `cmrMetadataFormat` config property instead and re-deploy
```

Comment:
I know this isn't part of your change, but this line is incorrect. `cmrMetadataFormat` is not used in `post-to-cmr`. It is only used in our fake processing task to tell the processing task what type of metadata file to generate for the tests.

Reply:
I actually did add that CHANGELOG line a previous PR because `cmrMetadataFormat` is used in `post-to-cmr` now. It was introduced by me:

https://github.com/nasa/cumulus/pull/824/files#diff-f18b744c47e44af9b04d5b9be7c32ff0R24

The reason why I needed the `cmrMetadataFormat`in the granule output from`post-to-cmr` was so that I can easily determine later (e.g. integration tests) whether I need to get online resources for UMM-G or ECHO10:

https://github.com/nasa/cumulus/pull/824/files#diff-6220467a3455e396ebf9431848b65712R337

For UMM-G, `cmrMetadataFormat` is specified as workflow meta and it can be version 1.4 or 1.5. So I thought the easiest way get that value to `post-to-cmr` was to pass it in as workflow config. Maybe there's a better way to go about this though?

Happy to pair on this if you want to walk through it.

Reply:
Ah ok, my concern here is we are using 2 different ways to determine what the CMR file type is and what version of UMM. Are we passing this in from move granules or having the user pass it in? I will approve this PR, but I think this is something we need to revisit.